app/controllers/api/messaging.php

```diff
@@ -2596,7 +2596,7 @@
     ->param('targets', [], new ArrayList(new UID()), 'List of Targets IDs.', true)
     ->param('cc', [], new ArrayList(new UID()), 'Array of target IDs to be added as CC.', true)
     ->param('bcc', [], new ArrayList(new UID()), 'Array of target IDs to be added as BCC.', true)
-    ->param('attachments', [], new ArrayList(new CompoundUID()), 'Array of compound bucket IDs to file IDs to be attached to the email.', true)
+    ->param('attachments', [], new ArrayList(new CompoundUID()), 'Array of compound ID strings of bucket IDs and file IDs to be attached to the email. They should be formatted as <BUCKET_ID>:<FILE_ID>.', true)
```

Comment:
probably need this for create and update push too...and why isn't this param in update email?? 😬

Reply:
+1, @stnguyen90 can you take the lead on fixing this?

Reply:
Created https://github.com/appwrite/appwrite/issues/7883 for me to work on